README.md

```diff
@@ -32,11 +32,19 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+Inside your '.env' file you will need to have the following keys:
+TRELLO_TOKEN
+TRELLO_KEY
+SECRET_KEY
+BOARD_ID
+
+You need to create an account and generate an API key and token. You will store your token and key values in your .env file under TRELLO_TOKEN and TRELLO_KEY.
+
 ## Running the App
 
 Once the all dependencies have been installed, start the Flask app in development mode within the Poetry environment by running:
 ```bash
-$ poetry run flask run
+$ poetry run python
```

Comment:
That would just open a Python REPL (interactive terminal).

The `poetry run flask run` instruction was correct.